### DISCUSSION TITLE
Is there a way to have expanded output by default?

### DISCUSSION THREAD

**Discussion by odanoburu** [marimo-team/marimo#2765] [Q&A]
Specially for markdown cells, it would be useful to have this option. When I write something, I don't want the reader to have to scroll the cell to read it.

![image](https://github.com/user-attachments/assets/8f0a70f9-4307-4f3a-b22f-13d2812f3c35)

**Comment by mscolnick** [comment]
Not currently, i think this would be a good default for markdown though at the very least. And secondly, we can potentially add this a CellConfiguration that gets serialized/saved

**Reply by odanoburu** [reply]
That would be nice! Should I open an issue?

**Reply by mscolnick** [reply]
@odanoburu, yes please, thanks!
